### DISCUSSION TITLE
Doodba postgres odoo initialization bug or feature

### DISCUSSION THREAD

**Discussion by ap-wtioit** [Tecnativa/doodba#561] [General]
Because we are setting `POSTGRES_DB` to `*dbname` at first start the postgres image is [creating an empty database](https://github.com/docker-library/docs/blob/master/postgres/README.md#postgres_db) with dbname before Odoo is starting. Which prevents odoo from automatically initialising the db in [_create_empty_database](https://github.com/odoo/odoo/blob/16.0/odoo/service/db.py#L106) and forces us to use something like `docker-compose run --rm odoo odoo -i base --stop-after-init` to initialize the db.

Also this way the extensions (pg_trgm, unaccent) normally created by Odoo are not created in this initialization procedure in doodba.

Should we fix this or has it been implemented to long and do we leave it as a "feature"?

**Comment by pedrobaeza** [comment]
Well, if you find a mechanism for avoiding the problem/feature, please propose the PR.

**Comment by xavivars** [comment]
What would be then the process to create a DB in prod?

First start everything (`docker-compose -f prod.yml up -d`) and then do something like `docker-compose run --rm odoo odoo -i base --stop-after-init` to reinitialize the db?

**Reply by pedrobaeza** [reply]
Flip both commands.
